packages/react-instantsearch-native/.babelrc

```diff
@@ -0,0 +1,85 @@
+{
+  "env": {
```

Comment:
Since we have the same .babelrc file everywhere, should we make it a .babelrc.js and have a single one that is required from those files? May or may not be relevant.

Reply:
we can only do .js in v7 of babel I think

Reply:
Indeed it's only available in Babel 7 (still in beta). But sill it depends if we want to keep the spirit of self-contained package. We can move the `.babelrc` at the top level (for Babel & Rollup at least) but it means that the child packages will depend on the top level one. For now I choose the most simplest way to implement it, the duplication. It's only the first iteration, we can choose to move all the configuration at the top in the next iteration. WDYT?

Reply:
I'd even prefer to change it to a shared _actual_ dependency like `kcd-scripts`, it can be in the same monorepo, that way it avoids any issues with duplication. WDYT?

Reply:
Yes indeed might be a solution.